airflow/www/static/js/index.js

```diff
@@ -0,0 +1,21 @@
+/**
```

Comment:
Should probably call this "base" or "common" or something - "index" is a unmeaningful to work out what might be in it.

Reply:
I picked 'index' only because that's the file name that the webpack [docs on Libraries](https://webpack.js.org/guides/author-libraries/#authoring-a-library) used. I'm guessing there's a way to override that - I'll dig in a bit more and see if I can use `base.js` instead.

Reply:
Not that important actually.